packages/dom/test/unit/lib/ElementAssertion.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { AssertionError, expect } from "@assertive-ts/core";
+import { render } from "@testing-library/react";
+import { ReactElement } from "react";
+
+import { ElementAssertion } from "../../../src/lib/ElementAssertion";
+
+function TestComponent(): ReactElement {
+  return (
+    <div>
+      <button>click me</button>
+    </div>
+  );
+}
+describe("[Unit] ElementAssertion.test.ts", () => {
+  describe(".toBeInTheDocument", () => {
+    context("when the element is in the document", () => {
+      it("returns the assertion instance", async () => {
+        const { findByRole } = render(<TestComponent />);
+        const button = await findByRole("button", {
+          name: "click me",
+        });
+        const test = new ElementAssertion(button);
+
+        expect(test.toBeInTheDocument())
+          .toBeEqual(test);
+
+        expect(() => test.not.toBeInTheDocument())
+          .toThrowError(AssertionError)
+          .toHaveMessage("Expected the element to NOT be in the document");
+      });
+    });
+    context("when the element is not in the document", () => {
+      it("Throws an assertion error", () => {
```

Comment:
```suggestion
      it("throws an assertion error", () => {
```

Reply:
Resolved in https://github.com/stackbuilders/assertive-ts/pull/129/commits/64c43ae6241f2e3e1e4b4ac13818abbeb6e7d4c9